### ISSUE TITLE
doc: Inaccurate/Incomplete "Unload/Disable AppArmor" documentation?

### ISSUE THREAD

**Issue by teward** [canonical/ubuntu-server-documentation#105]
The AppArmor documentation stated with the SystemD option to disable AppArmor and unload/disable the Kernel module doesn't appear to work - per user statements here: https://askubuntu.com/questions/1536716/why-cant-i-disable-apparmor-on-24-04-1?noredirect=1#comment2703457_1536716

However, lifting from the original documentation on AppArmor on the Wiki (ancient! - https://help.ubuntu.com/community/AppArmor#Disable_AppArmor_framework) it includes how to disable AppArmor in GRUB's defaults. According to the user in the Ask Ubuntu statement, the SystemD approach did not disable the AppArmor kernel module nor unload it even on reboot with the service disabled.

It took revisions to /etc/default/grub per the original AppArmor wiki page to properly disable the AppArmor kernel modules.

This may be a bug in the SystemD task, or something that is overlooked in the Server documentation, but needs reviewed for accuracy - if it's not accurate here, then the documentation needs updated; if it's a bug in the SystemD tasks then Kernel and Security teams need to be aware of the broken service load/unload issues.

---
Document: how-to/security/apparmor.md

**Comment by cpaelzer** [comment]
You are right, it is built-in anyway so loading modules or not isn't the point.
I think it is a bug in the documentation outdated from when it was module only loaded that way (assumption, not knowledge).

1. We should also split disabling profiles from disabling apparmor in https://documentation.ubuntu.com/server/how-to/security/apparmor/#disabling-or-re-enabling-a-profile to be two sections.
2. Friendly steal the pre-amble from https://help.ubuntu.com/community/AppArmor#Disable_AppArmor_framework to better not do so.
3. We should drop any "module" mentioning since it is compiled in
4. and mention the disabling via the option (which will be picked up on boot by the kernel built-in or module if it is one) by setting this in grub (we can take that from https://help.ubuntu.com/community/AppArmor#Disable_AppArmor_framework as you said)
